middleware/auth.js

```diff
@@ -11,8 +11,6 @@ const auth = async (req, resp, next) => {
             token = token.split(' ')[1]
             const data = jwt.verify(token, jwt_key)
             req.userId = data.id
```

Comment:
Use only camelCase convention. Don't use snake_case in js.

Reply:
ok next time use camelCase